src/graphql/main/lexicographic-sort-schema.graphql

```diff
@@ -46,6 +46,33 @@ type AuthTokenPayload {
   errors: [Error!]!
 }
 
+type BTCWallet implements Wallet {
```

Comment:
not clear why this has been added to the schema

Reply:
Its a potential future proofing as we will very likely have a different type for `FiatWallet` or `USDWallet`

Reply:
I know but I don't know why this appear here in this PR (this is the auto generated file), as I have not changed anything regarding the BTCWallet interface on the graphql layer AFAIK